src/win/tcp.c

```diff
@@ -556,6 +556,8 @@ static void uv_tcp_queue_read(uv_loop_t* loop, uv_tcp_t* handle) {
 
 int uv_tcp_listen(uv_tcp_t* handle, int backlog, uv_connection_cb cb) {
   uv_loop_t* loop = handle->loop;
+  struct sockaddr saddr;
```

Comment:
This won't work for IPv6 addresses. Use `struct sockaddr_storage` and cast it to `struct sockaddr`.